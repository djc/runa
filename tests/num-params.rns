def foo(i: i32) -> i32:
	return i

def main(name: &str, args: &array[str]) -> i32:
	foo(3, 4)
	return 0
