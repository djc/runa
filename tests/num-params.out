tests/num-params.rns [5.5]: arguments (int, int) cannot be passed as (i32)
    foo(3, 4)
       ^
